utils/command_converter.py

```
@@ -0,0 +1,228 @@
+import logging
+import platform
+import os
+import re
+from typing import Optional, Dict, Any
+from pathlib import Path
+import asyncio
```

Flake8 (7.2.0)

[error] 'asyncio' imported but unused

(F401)